python/pacemaker/_cts/tests/remotestonithd.py

```diff
@@ -9,12 +8,24 @@
 
 
 class RemoteStonithd(RemoteDriver):
+    """ A concrete test that fails and fences the node hosting a Pacemaker
+        Remote connection resource
```

Comment:
it fails the connection resource but the remote node gets fenced (not the connection host)